tests/benchmarks/garage/tf/algos/benchmark_ppo.py

```diff
@@ -264,49 +247,3 @@ def run_garage_tf(env, seed, log_dir):
         dowel_logger.remove_all()
 
         return tabular_log_file
-
-
-def run_baselines(env, seed, log_dir):
```

Comment:
Do we want to delete this? This should be the only place we could locate the baselines code.

Reply:
It looks like the openai.baseline version is not support Tensorflow 2.0.